From bdd91e29225b44a1a13e670dc4cecfb58f241d28 Mon Sep 17 00:00:00 2001
From: Davide Pesavento <pesa@gentoo.org>
Date: Sun, 7 Aug 2016 12:14:57 +0200
Subject: [PATCH] Disable failing torrent_parse tests

For unknown reasons, some test torrents are missing from the tarball.
Skip the corresponding test cases to avoid spurious failures.
---
 test/test_torrent_parse.cpp | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/test/test_torrent_parse.cpp b/test/test_torrent_parse.cpp
index 663f341..2de6fb5 100644
--- a/test/test_torrent_parse.cpp
+++ b/test/test_torrent_parse.cpp
@@ -49,13 +49,10 @@ using namespace libtorrent;
 test_torrent_t test_torrents[] =
 {
 	{ "base.torrent" },
-	{ "empty_path.torrent" },
 	{ "parent_path.torrent" },
 	{ "hidden_parent_path.torrent" },
 	{ "single_multi_file.torrent" },
 	{ "slash_path.torrent" },
-	{ "slash_path2.torrent" },
-	{ "slash_path3.torrent" },
 	{ "backslash_path.torrent" },
 	{ "url_list.torrent" },
 	{ "url_list2.torrent" },
@@ -75,8 +72,6 @@ test_torrent_t test_torrents[] =
 	{ "root_hash.torrent" },
 	{ "empty_path_multi.torrent" },
 	{ "duplicate_web_seeds.torrent" },
-	{ "invalid_name3.torrent" },
-	{ "symlink1.torrent" },
 };
 
 struct test_failing_torrent_t
-- 
2.9.2

